benchmarking/smartseq2/merge_expression_matrix/run_merge_quants.wdl

```diff
@@ -0,0 +1,38 @@
+task MergeTPM {
+  String target_dir
+  String run_name
+  String output_name
+  String uuid
+  String value_name
+  command {
```

Comment:
Be consistent about spacing between blocks; suggest we use what Jose's style guide suggests.